src/shame.js

```diff
@@ -112,3 +112,55 @@ export const shamefullyAddTableColumns = (graphDiv, {traceIndexes, update}) => {
     update['header.values'] = null;
   }
 };
+
+export const shamefullyCreateSplitStyles = (
+  graphDiv,
+  attr,
+  traceIndex,
+  splitTraceGroup
+) => {
+  if (!Array.isArray(splitTraceGroup)) {
+    splitTraceGroup = [splitTraceGroup]; // eslint-disable-line
+  }
+
+  let indexOfSplitTransform = null;
+
+  graphDiv.data[traceIndex].transforms.filter((t, i) => {
```

Comment:
why filter?

Reply:
lol, just didn't want to write a for loop.. i could write a for loop

Reply:
forEach?

Reply:
our linter won't complain, but generally function inside filter should return something...

Reply:
yeah you're right, changing